### PR TITLE
GitHub Actions migration

### PR DESCRIPTION
**Pull request by guillemcortes**
TODO list:

- [x] Change docs and contributing sections. Specify the python supported versions and update contribution guidelines
- Paths normalization with os.path.normpath() so they work with windows. --》To be done in another PR.
- [x] Review dependencies in general (readthedocs, environment-lint, environment-ci, etc)

### CODE REVIEW

File: .github/environment-ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: mirdata-dev-py310
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python==3.10
```
Comment (guillemcortes) [inline .github/environment-ci.yml:6]: remove this

File: .github/environment-ci.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: mirdata-dev-py310
```
Comment (guillemcortes) [inline .github/environment-ci.yml:1]: specify maximum python3.7 packages versions

File: tests/datasets/test_ikala.py [hunk 3]
```diff
@@ -92,10 +93,25 @@ def test_to_jams():
...
+    for i, f0 in enumerate(f0s):
+        assert "frequency" in f0.value
+        assert "index" in f0.value
+        assert "voiced" in f0.value
+        assert math.isclose(
+            f0.value["frequency"], expected_f0s[i]["frequency"], rel_tol=1e-12
+        )
+        assert f0.value["index"] == expected_f0s[i]["index"]
+        assert f0.value["voiced"] == expected_f0s[i]["voiced"]
+    # assert [f0.value for f0 in f0s] == [
```
Comment (guillemcortes) [inline tests/datasets/test_ikala.py:109]: add a comment stating that this is how it was before

**Review by genisplaja — APPROVED** [review]
LGTM! We can try to merge and see if we need to fix something later, as we did in soundata :)

**Pull request merged**